### ISSUE TITLE
Snake high score placeholder not working.

### ISSUE THREAD

**Issue by ZizzyZizzy** [Maximvdw/ArcadeBoard-API#6]
Using DynamicSigns. High score never shows.

` 

>  snek1:
>     frames:
>     - '&2&lHigh Score'
>     interval: 2000
>   snek2:
>     frames:
>     - '&f&l+---------+'
>     interval: 2000
>   snek3:
>     frames:
>     - '&b{arcadeboard_snake_statistic_best_score}'
>     interval: 200

`

**Comment by ZizzyZizzy** [comment]
Seems like the SQL data is not in the same format as the others. Perhaps that's the issue?
```
# sqlite3 ArcadeBoard.db
SQLite version 3.7.17 2013-05-20 00:56:22
Enter ".help" for instructions
Enter SQL statements terminated with a ";"
sqlite> .tables
player_game_data  players           plugin_settings
sqlite> select * from player_game_data;
1|Snake|1|best_score|390|1573674416517
2|Stack|1|STATISTIC_BEST_SCORE|37.0|1573674717181
3|Tetris|1|STATISTIC_BEST_SCORE|1700.0|1573675086140
4|Stack|1|STATISTIC_BEST_SCORE|9.0|1573685054266
5|Stack|1|STATISTIC_BEST_SCORE|12.0|1573692753362
6|Stack|2|STATISTIC_BEST_SCORE|5.0|1573698566652
7|Snake|2|best_score|40|1573698600856
8|Tetris|1|STATISTIC_BEST_SCORE|2700.0|1573702675537
9|Tetris|1|STATISTIC_BEST_SCORE|5800.0|1573745800596
10|Pong|2|STATISTIC_VICTORIES|1.0|1573746852198
11|Pong|1|STATISTIC_DEFEATS|1.0|1573746852198
12|Stack|1|STATISTIC_BEST_SCORE|29.0|1573747035520
````

**Comment by ZizzyZizzy** [comment]
Confirmed! Once I changed the 'key' column in the database to 'STATISTIC_BEST_SCORE' instead of 'best_score', the sign properly displays the high score for Snake.

```
sqlite> select * from player_game_data;
1|Snake|1|STATISTIC_BEST_SCORE|390|1573674416517
2|Stack|1|STATISTIC_BEST_SCORE|37.0|1573674717181
3|Tetris|1|STATISTIC_BEST_SCORE|1700.0|1573675086140
4|Stack|1|STATISTIC_BEST_SCORE|9.0|1573685054266
5|Stack|1|STATISTIC_BEST_SCORE|12.0|1573692753362
6|Stack|2|STATISTIC_BEST_SCORE|5.0|1573698566652
7|Snake|2|STATISTIC_BEST_SCORE|40|1573698600856
8|Tetris|1|STATISTIC_BEST_SCORE|2700.0|1573702675537
9|Tetris|1|STATISTIC_BEST_SCORE|5800.0|1573745800596
10|Pong|2|STATISTIC_VICTORIES|1.0|1573746852198
11|Pong|1|STATISTIC_DEFEATS|1.0|1573746852198
12|Stack|1|STATISTIC_BEST_SCORE|29.0|1573747035520
```

That should mean the source code needs to be changed in the Snake jar, correct?

**Comment by ZizzyZizzy** [comment]
@Maximvdw Also, see the output from the sqlite DB above. It's writing a new row for the most recent new "BEST_SCORE" data instead of updating the existing best score for that player.

Furthermore, when you restart the server, the plugin acts like all the scores have been reset and will not display a new best score until you play again, at which point ANOTHER row is added to the sqlite database. Notice the "stack" data for player "1" in that output. That was after exactly 4 game plays and 4 server restarts, and the DB is showing a separate row for each game play.

tl;dr Restarting the server is exactly like the power plug on the arcade cabinet was pulled and the backup battery for the CMOS was drained. *poof* no more high scores.

**Comment by Maximvdw** [comment]
fixed
